src/Data/List/Relation/Unary/First.agda

```diff
@@ -106,3 +105,4 @@ module _ {q} {Q : Pred A q} where
   toAny : ∀ {p} {P : Pred A p} → First P Q ⊆ Any Q
   toAny [ qx ]     = here qx
   toAny (_ ∷ pqxs) = there (toAny pqxs)
+
```

Comment:
```suggestion
```
@jmougeot Either you or your editor seems to be inserting these additional blank lines (not just this PR). While I'm not sure that `style-guide` mandates what should be done in these situations, I don't think PRs should change existing whitespace unless for very good reasons